BEGINNERS.md

```diff
@@ -54,9 +54,11 @@ follow the [Vagrant setup instructions](SETUP_VAGRANT.md).
 **This is the best and easiest option for all systems including
 GNU/Linux, BSD, Mac OS X, and Windows.**
 
-As an alternative option, we also have instructions for local installation.
-[Debian-based instructions](SETUP_DEBIAN.md) work for Debian or Ubuntu or
-related, and our general [GUIDE](GUIDE.md) has some notes for other systems.
+As an alternative option, we also have instructions for local
```

Comment:
Oh, it's Emacs notation

Alt+X
Type in `fill-paragraph`

It's an Emacs function that does line wrapping. It's actually bound to
`M-q`, but that's not important. The important thing is that the line
wrapping was automatic, I didn't do it on purpose.

Peter Harpending

On Fri, May 22, 2015 at 05:33:36PM -0700, Aaron Wolf wrote:
> > @@ -54,9 +54,11 @@ follow the [Vagrant setup instructions](SETUP_VAGRANT.md).
> >  **This is the best and easiest option for all systems including
> >  GNU/Linux, BSD, Mac OS X, and Windows.**
> >  
> > -As an alternative option, we also have instructions for local installation.
> > -[Debian-based instructions](SETUP_DEBIAN.md) work for Debian or Ubuntu or
> > -related, and our general [GUIDE](GUIDE.md) has some notes for other systems.
> > +As an alternative option, we also have instructions for local
> 
> I don't know what that is, but it's better to fit complete sentences
> on their own lines when possible, and makes diffs and other things
> cleaner. Try to use your own judgement to make lines make sense when
> you can.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/snowdriftcoop/snowdrift/pull/310/files#r30939736